sdk/node/src/connection.js

```diff
@@ -34,6 +81,8 @@ class Connection {
       body = {}
     }
 
+    const snakeBody = snakeize(body) // Ssssssssssss
```

Comment:
Love this comment, but I would like to expand on it to explain why this is happening.